components/automate-dex/web/templates/login.html

```diff
@@ -1,7 +1,7 @@
 <!-- rework of coreos login.html-->
 <!-- rendered using https://golang.org/pkg/html/template/-->
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
valid - possibly better to us `en-US`.  This declares United States English.  Open to debate.

Reply:
automate-ui uses en, so i. think lets keep them consistent for now.